mtags/src/main/scala/scala/meta/internal/docstrings/ScaladocParser.scala

```diff
@@ -1063,8 +1063,8 @@ object ScaladocParser {
 
     /* INLINES */
 
-    val OPEN_TAG = "^<([A-Za-z]+)( [^>]*)?(/?)>$".r
-    val CLOSE_TAG = "^</([A-Za-z]+)>$".r
+    val OPEN_TAG: Regex = "^<([A-Za-z]+)( [^>]*)?(/?)>$".r
+    val CLOSE_TAG: Regex = "^</([A-Za-z]+)>$".r
```

Comment:
these are other examples where I don't see a great value.

Reply:
Sure, but this particular is not a major regression either IMO. Once you get used to always having public types I personally find it quite helpful even in cases like this, now you don't even need to squint to observe the trailing `.r`